db/migrate/20230713214036_data_model.rb

```diff
@@ -0,0 +1,10 @@
+class DataModel < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :post_counter
+    end
```

Comment:
- [ ] Great job working on the users migration but I think you forgot to create the rest of the migration displayed in the ERD so kindly create a migrations for posts, comments and likes tables since it is mentioned in the requirements to create a schema for all the tables shown in the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) 🚀💯💪🏻💪🏻


- [ ] Kindly don't forget to add in All columns that are foreign keys should have a corresponding index. 🙏🏻




![image](https://github.com/Tayyab-Kamboh/blog-app/assets/58553711/ea39466a-49a4-4131-b40c-92dada8e49de)

Reply:
Yes actually, I was a bit confused about this as there are multiple tables in the ERD diagram for now I have added the user table, do I add the others now or are they to be added in the upcoming projects?